source/mongorocks.rst

```diff
@@ -8,6 +8,12 @@ MongoRocks
 based on the RocksDB_ key-value store optimized for fast storage.
 It is developed by Facebook and designed to handle write-intensive workloads.
 
+.. warning:: *MongoRocks* is deprecated in 3.6 and will be removed in the
+   next major version! Feature compatibility version for MongoRocks
+   is limited to 3.4 so 3.6 functionality cannot be used. Read/write
+   concern majority is not safe to use as there are a number of
+   failing tests for those.
```

Comment:
Please change to;
is limited to 3.4, so 3.6 functionality cannot be used. Read/write concern majority is not safe to use.